plugins/inputs/kinesis_consumer/kinesis_consumer.go

```diff
@@ -66,8 +72,15 @@ type (
 
 const (
 	defaultMaxUndeliveredMessages = 1000
+	gzipDecompression             = "gzip"
+	noDecompression               = "none"
+	zlibDecompression             = "zlib"
 )
 
+var supportedDecompressionAlgorithms = []string{gzipDecompression, noDecompression, zlibDecompression}
```

Comment:
Please move this to `Init()` instead of declaring a global variable and directly define the list instead of using the constants.